guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,170 @@
+baseurl=https://yum.theforeman.org/pulpcore/3.16/el8/$basearch/
+gpgkey=https://yum.theforeman.org/pulpcore/3.16/GPG-RPM-KEY-pulpcore
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+endif::[]
+
+[leapp-foreman-plugins]
+name={ProjectVersion}
+baseurl=https://yum.theforeman.org/plugins/3.2/el8/$basearch
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+module_hotfixes=1
+
+[leapp-foreman-client]
+name=Foreman client 3.2
+baseurl=https://yum.theforeman.org/client/3.2/el8/$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman-client
+
+[leapp-puppet7]
+name=Puppet 7 Repository el 8 - $basearch
+baseurl=http://yum.puppetlabs.com/puppet7/el/8/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet7-release
+       file:///etc/pki/rpm-gpg/RPM-GPG-KEY-2025-04-06-puppet7-release
+enabled=1
+gpgcheck=1
+----
+
+* If you are using Puppet 6 instead of Puppet 7, replace the `7` with a `6` in the `leapp-puppet7` entry.
+
+* You need a Puppet repository for the Puppet agent that the installer is using.
+For more information, see link:https://docs.theforeman.org/nightly/Managing_Configurations_Puppet/index-katello.html[Configuring Hosts Using Puppet].
+
+. We do not support Enterprise Linux 8 installations with EPEL 8 enabled, so remove `epel-release`:
++
+----
+# yum remove epel-release
+----
+
+. Remove `centos-release-scl` and `centos-release-scl-rh` repositories:
++
+----
+# yum remove centos-release-scl centos-release-scl-rh
+----
+
+. Let LEAPP analyze your system:
++
+----
+# leapp preupgrade
+----
+The first run is expected to fail but report issues.
+Continue to the next step for remediation.
+
+. Fix issues found by LEAPP (check `/var/log/leapp/leapp-report.txt` for details):
++
+----
+# rmmod pata_acpi
+# echo PermitRootLogin yes | tee -a /etc/ssh/sshd_config
+# leapp answer --section remove_pam_pkcs11_module_check.confirm=True
+----
+
++
+The `preupgrade` might fail with a dependency resolution error such as:
++
+----
+* "package rubygem-fx-0.5.0-2.el8.noarch requires rubygem(railties) >= 4.0.0, but none of the providers can be installed"
+* "package rubygem-railties-6.0.4.7-1.el8.noarch requires rubygem(thor) < 2.0, but none of the providers can be installed"
+----
++
+If this happens, do the following to clean up packages that cannot automatically upgrade (`rubygem(thor)` and `rubygem(railties)` in the example above):
++
+----
+# yum remove rubygem-thor rubygem-railties
+----
++
+Check the output if it does not contain any local packages.
++
+Remove `rubygem-thor`:
++
+----
+# yum remove rubygem-thor
+----
+. Ensure `leapp preupgrade` has no issues.
+
+. Run:
++
+----
+# leapp upgrade
+----
+. Reboot the system.
+
+. After the system reboots, a live system conducts the upgrade, reboots to fix SELinux labels, then reboots into the final Enterprise Linux 8 system.
+. After the system reboots, LEAPP finishes the upgrade, watch it with:
```

Comment:
starting two points with "After the system reboots" sounds odd to me, but I am not sure how one could re-word this better.

Reply:
@evgeni I made it easy. Just removed "After the system reboots" and it still makes sense.